content/docs/testing-recipes.md

```diff
@@ -80,13 +63,13 @@ export default function User(props) {
     fetchUserData(props.id);
   }, [props.id]);
   return user === null ? (
-    "loading..."
+    <div>loading...</div>
   ) : (
     <details>
-      <summary>{user.name}</summary>
-      <strong>{user.age}</strong> years old
+      <summary aria-label="name">{user.name}</summary>
```

Comment:
Without the labels, the data would be difficult to understand for people using screen readers. Honestly, in a real app you'd probably have real labels and/or icons for this data (which would also have labels). So I'm adding these here so the tests can resemble what this kind of thing would look like in the real world.